Source/WebCore/html/shadow/DateTimeEditElement.cpp

```diff
@@ -404,6 +404,15 @@ DateTimeFieldsState DateTimeEditElement::valueAsDateTimeFieldsState() const
     return dateTimeFieldsState;
 }
 
+bool DateTimeEditElement::editableFieldsHaveValues() const
+{
+    for (auto& field : m_fields) {
+        if (!field->isDisabledFormControl() && field->hasValue())
```

Comment:
The fields are just `HTMLDivElement`, so `isDisabledFormControl()` is always going to be false?

Reply:
Removed the `isDisabledFormControl()`, this was copy pasta from Chromium